package.json

```diff
@@ -68,7 +68,7 @@
   "resolutions": {
     "connected-next-router/react-redux": "8.0.7",
     "react-beautiful-dnd/react-redux": "8.0.7",
-    "@prismicio/types-internal": "2.4.1"
+    "@prismicio/types-internal": "2.5.0-alpha.4"
```

Comment:
@angeloashmore When I tried removing this, things seemed to work, but all the CI tests failed.

Reply:
@xrutayisire helped me track this down today. The reason the CI tests were failing is because `@prismicio/editor-fields` has a dependency on `@prismicio/types-internal`.

Reply:
Ah, okay. In that case, we should update `@prismicio/editor-fields` to use the latest `@prismicio/types-internal`. We should avoid `resolutions` as much as possible since it could break packages.